### PR TITLE
ECN Counts; last fix

### PR DESCRIPTION
**Pull request by martinthomson**
This one slipped the net.

### CODE REVIEW

File: rfc9000.md [hunk 1]
```diff
@@ -5687,7 +5687,7 @@ ECN Counts {
 ~~~
 {: #ecn-count-format title="ECN Count Format"}
 
-The three ECN Counts are:
+The three fields in ECN Counts are:
```
Comment (martinthomson) [inline rfc9000.md:5690]: I could also do

```suggestion
The three fields in the ECN Counts field are:
```

**Review by DavidSchinazi — APPROVED** [review]
Either way is fine by me. Just ship it already :P

File: rfc9000.md (hunk 1 above)
Comment (martinthomson) [reply rfc9000.md:5690]: Or
```suggestion
The ECN count fields are:
```

File: rfc9000.md [hunk 2]
```diff
@@ -5674,7 +5674,7 @@ connection error of type FRAME_ENCODING_ERROR.
 The ACK frame uses the least significant bit of the type value (that is, type
 0x03) to indicate ECN feedback and report receipt of QUIC packets with
 associated ECN codepoints of ECT(0), ECT(1), or ECN-CE in the packet's IP
-header.  ECN Counts are only present when the ACK frame type is 0x03.
+header.  The ECN Counts field is only present when the ACK frame type is 0x03.
```
Comment (martinthomson) [inline rfc9000.md:5677]:
```suggestion
header.  ECN counts are only present when the ACK frame type is 0x03.
```

**Review by janaiyengar — APPROVED** [review]
Thanks, Martin!

**Pull request merged**